.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,74 @@
+on:
+  workflow_run:
+    workflows:
+      - Build [Docs]
+    types:
+      - completed
```

Comment:
does `completed` mean successfully completed?

Reply:
Nope, just that it's finished regardless of state.

Reply:
got it - cool